.github/workflows/publishVSCode.yml

```diff
@@ -67,10 +64,10 @@ jobs:
         with:
           ref: 'main'
           token: ${{ secrets.IDEE_GH_TOKEN }}
-      - uses: actions/download-artifact@v3
-        with:
-          name: ${{ inputs.version }}
-          path: ./extensions
+      - name: downloadExtensionsFromRelease
+        run: | 
+          mkdir ./extensions
+          gh release download v${{ env.PUBLISH_VERSION }} -D ./extensions
```

Comment:
Using the gh cli to download the vsix files attached to the release which are generated in the release workflow.

Reply:
Ohhh I like this